### PR TITLE
Add alt text to the logo on primaryAuth.

### PR DESCRIPTION
**Pull request by alexeystadnikov-okta**
Resolves OKTA-104130

okta-core PR: https://github.com/okta/okta-core/pull/17982

screen shot 2016-12-13 at 5 24 05 pm

@mauriciocastillosilva-okta 
@lboyette-okta 

Please, review.

### CODE REVIEW

File: src/models/Settings.js [hunk 1]
```diff
@@ -46,6 +46,7 @@ function (Okta, Errors, BrowserFeatures, Util, Logger, config) {
       'recoveryToken': ['string', false, undefined],
       'stateToken': ['string', false, undefined],
       'username' : ['string', false],
+      'logoText' : ['string', false],
```
Comment (mauriciocastillosilva-okta) [inline src/models/Settings.js]: nit: can we move this to line 61? That's where the ```logo``` is

**Comment by mauriciocastillosilva-okta** [comment]
Also, please change the commit message to follow the convention https://github.com/okta/okta-signin-widget/blob/master/CONTRIBUTING.md

File: src/models/Settings.js (hunk 1 above)
Comment (alexeystadnikov-okta) [inline src/models/Settings.js]: We need `logo` (contains path to image) and `logoText` (will be org.name for the alt attribute).

**Comment by alexeystadnikov-okta** [comment]
@mauriciocastillosilva-okta 
Changes are very small and don't contain any logic. Not sure what can we test here.

File: src/models/Settings.js (hunk 1 above)
Comment (alexeystadnikov-okta) [inline src/models/Settings.js]: Done

**Pull request merged**